server/pages/index.js

```diff
@@ -388,7 +390,15 @@ function setUpCSP( req, res, next ) {
 		'style-src': [ "'self'", '*.wp.com', 'https://fonts.googleapis.com' ],
 		'form-action': [ "'self'" ],
 		'object-src': [ "'none'" ],
-		'img-src': [ "'self'", '*.wp.com', 'https://www.google-analytics.com' ],
+		'img-src': [
+			"'self'",
```

Comment:
Is it possible to add `data` somehow?
I got the following csp report sent on `/log-in`:
```
blocked-uri:"data"
document-uri:"https://wordpress.com/log-in"
...
violated-directive:"img-src"
```

Reply:
Will add.